frontend/src/components/RideModal/Pages/Driver.tsx

```diff
@@ -1,42 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { ModalPageProps } from '../../Modal/types';
 import styles from '../ridemodal.module.css';
 import { Label, Input, Button } from '../../FormElements/FormElements';
-import { useEmployees } from '../../../context/EmployeesContext';
+import { useReq } from '../../../context/req';
 
 const DriverPage = ({ onBack, onSubmit, formData }: ModalPageProps) => {
   const { register, handleSubmit, formState } = useForm({
-    defaultValues: { driver: formData?.driver ?? '' },
+    defaultValues: {
+      driver: formData?.driver ?? '',
+    },
   });
   const { errors } = formState;
-  const { drivers } = useEmployees();
+  const { withDefaults } = useReq();
+  const [refreshed, setRefreshed] = useState(false);
+
+  const { date, pickupTime: startTime, dropoffTime: endTime } = formData!;
   type DriverOption = { id: string; firstName: string; lastName: string };
-  const driverOptions: DriverOption[] = [
-    { id: 'None', firstName: 'None', lastName: '' },
-  ].concat(drivers);
+  const [availableDrivers, setAvailableDrivers] = useState<DriverOption[]>([]);
```

Comment:
we want to preserve the "None" option from the original `driverOptions` variable, since admins might not want to assign a driver yet even if there are ones available

Reply:
Okay, I will do that.
Am I understanding this correctly? It seems adding the "None" option will make the card never displays the word "none".

Reply:
I think just the none option is fine, so the "none" word isnt needed

Reply:
okay